core/io/image.cpp

```diff
@@ -3464,6 +3468,7 @@ void Image::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("compress_from_channels", "mode", "channels", "astc_format"), &Image::compress_from_channels, DEFVAL(ASTC_FORMAT_4x4));
 	ClassDB::bind_method(D_METHOD("decompress"), &Image::decompress);
 	ClassDB::bind_method(D_METHOD("is_compressed"), &Image::is_compressed);
+	ClassDB::bind_static_method("Image", D_METHOD("is_format_compressed", "format"), &Image::is_format_compressed);
```

Comment:
I don't think we need to expose this to users.

Reply:
Agree, if needed it'll be asked about.